bin/generate_gene_counts.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import os
+from collections import OrderedDict
+
+
+def read_star_gene_cnts(sample, star):
+    """Read gene counts file from STAR output."""
+    sample_ids = {}
+    gene_ids = {}
+    with open(star) as in_tab:
+        for line in in_tab:
+            if line.startswith("ENSG"):
```

Comment:
We cannot be certain that every user is going do use the ENSMBL gene annotation.
Can you make this a bit more "agnostic"?

Reply:
Yup! Will do this in the coming days 👍